tests/test_logfire.py

```diff
@@ -3171,3 +3173,23 @@ def test_suppress_scopes(exporter: TestExporter, metrics_reader: InMemoryMetricR
             }
         ]
     )
+
+
+def test_logfire_span_records_exceptions_once():
+    # logfire.configure(send_to_logfire=False, console=False, additional_span_processors=[BatchSpanProcessor(exporter)])
+    n_calls_to_record_exception = 0
+
+    def patched_record_exception(*args: Any, **kwargs: Any) -> Any:
+        nonlocal n_calls_to_record_exception
+        n_calls_to_record_exception += 1
+
+        return record_exception(*args, **kwargs)
+
+    with patch('logfire._internal.tracer.record_exception', patched_record_exception), patch(
+        'logfire._internal.main.record_exception', patched_record_exception
+    ):
+        with pytest.raises(RuntimeError):
+            with logfire.span('foo'):
+                raise RuntimeError('error')
+
+    assert n_calls_to_record_exception == 1
```

Comment:
I confirmed this test fails on `main` and has two calls instead of one.

Reply:
It looks like the doubling was introduced by https://github.com/pydantic/logfire/pull/696